### PR TITLE
fmt: wrap comments

### PR DESCRIPTION
**Pull request by wischli**
# Description

* Adds and applies comment wrapping to format config
* Based on discussion in [DAO Slack](https://centrifugedao.slack.com/archives/C04H149SU5D/p1683015915469879)

# Checklist:

- [x] I have added Rust doc comments to structs, enums, traits and functions
- [x] I have made corresponding changes to the documentation
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: libs/test-utils/src/mocks/order_manager.rs [hunk 1]
```diff
@@ -205,8 +206,8 @@ pub mod pallet {
 			orders.amount += amount;
 			RedeemOrders::<T>::insert(investment_id, orders);
 
-			// NOTE: TrancheTokens NEVER leave the TEST_PALLET_ID account and hence we can keep them here and
-			//       need no transfer.
+			// NOTE: TrancheTokens NEVER leave the TEST_PALLET_ID account and hence we can
+			// keep them here and       need no transfer.
```
Comment (NunoAlexandre) [inline libs/test-utils/src/mocks/order_manager.rs:210]:
```suggestion
			// keep them here and need no transfer.
```

**Review by NunoAlexandre — COMMENTED** [review]
Just caught one weird spacing issue, the rest looks good!

File: libs/test-utils/src/mocks/order_manager.rs (hunk 1 above)
Comment (wischli) [reply libs/test-utils/src/mocks/order_manager.rs:210]: Well spotted Eagle Eye!

**Review by NunoAlexandre — APPROVED** [review]
❇️

**Review by lemunozm — DISMISSED** [review]

**Review by mustermeiszer — APPROVED** [review]

**Review by lemunozm — APPROVED** [review]

**Comment by lemunozm** [comment]
Do we have any way to ensure this is correct for new PRs in CI? (maybe in our docs job)

**Comment by wischli** [comment]
> Do we have any way to ensure this is correct for new PRs in CI? (maybe in our docs job)

Yes! The new rule is applied to our linting CI check `cargo fmt -- --check`. That's why the CI failed yesterday after just merging `main` without re-running `cargo fmt`.

**Review by mustermeiszer — APPROVED** [review]
Re-approve

**Pull request merged**